.github/workflows/lint-python.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
-            python-version: "3.8"
+            python-version: "3.11"
```

Comment:
Is there a particular reason for this direct jump to 3.11? Well, in the long run probably will ensure less changes as older versions of Python get deprecated...

Reply:
yeah, I just thought it was good to keep up-to-date with the latest python version

Reply:
alright why not :) Let's move up to 3.11!